lib/index.js

```diff
@@ -250,8 +250,9 @@ async function _getKeyRecord({id}) {
     return promise;
   }
 
-  promise = _getUnCachedKeyRecord({id});
-  KEY_RECORD_CACHE.set(id, promise);
+  promise = _getUncachedKeyRecord({id});
+  KEY_RECORD_CACHE.set(
+    id, promise, bedrock.config['ssm-mongodb'].keyRecordCache.maxAge);
```

Comment:
same question as in other usage, why is maxAge being passed in here rather than as part of the constructor to lru-cache?

Reply:
Fixed this.